package.json

```diff
@@ -1186,6 +1187,7 @@
     "vscode-languageserver-textdocument": "^1.0.12"
   },
   "devDependencies": {
+    "@anthropic-ai/sdk": "^0.25.2",
```

Comment:
Do we want to support both `@anthropic-ai/sdk` and `openai`? It adds extra dependencies to the project and we are not even planning to run these tests nightly or on schedule. How do you feel about keeping just `openai`, since this is something that we also use in compass?

Reply:
Sounds good to me, I'll remove anthropic